content/blog/zephyrs-summer-intern-experience-with-pulumi/index.md

```diff
@@ -0,0 +1,81 @@
+
+In addition, I absolutely love the various team-building games, especially the virtual escape room, and the pair-up chatting with other engineers in the team. These interesting events and relaxed atmosphere made me feel more welcomed and less stressful. I've been moved and realized it is okay to be imperfect and improve step by step. Despite having less coding experience, I was still welcomed and cared for by everyone at Pulumi.
+
+Moreover, I am so glad that I could work on my individual internship project: building the content management system(CMS) for Pulumi's website. I have taken control of the whole development phase of the Content Management System project. I was able to research and write up proposals, implement the project, plan the next step, and improve. In the end, the CMS was published and running. I have contributed back to the Netlify CMS, the provider for the CMS. I have compiled CMS as an [open-source example](https://github.com/pulumi/examples/tree/master/aws-ts-netlify-cms-and-oauth) and written blog posts on deploying the [CMS application]({{< relref "/blog/deploying-netlify-cms-on-aws" >}}) and its [OAuth Access Token Server]({{< relref "/blog/deploying-the-infrastructure-of-oauth-server-for-cms-app" >}}) to AWS.
+
+When I heard the CMS project would be a great help for the marketing team during our meetings, I felt indescribable happiness and proud of myself. Although far from perfect, I was finally able to prove myself with this unforgettable experience.
+
+The confidence is the greatest gift that Pulumi gave me during this Summer internship. I felt a lot more confident in entering the technology industry and pursue my career goal as a software engineer. Thanks to this internship experience, I have grown braver and stronger than before.
+
+## Technology Skills
+
+I have learned a lot of skills that I have never learned in the classrooms.
+
+- First of all, I learned a lot about Pulumi and cloud technology.
+  - Before my internship, I only read about that Pulumi provides the code for building infrastructure. At first, I thought the benefit would just be to reduce the amount of work to click through different settings on the provider's website. However, during the development process of CMS, I found writing infrastructure as code can do many more things than that. For example, I have implemented the Github Actions, which will deploy the infrastructure automatically when I push or merge a pull request. It saves time to upload and deploy again and prevent any manual error by automating the process. Also, if we want to completely remove the cloud settings, Pulumi enables us to do a simple `pulumi destroy` rather than going through cloud settings and worry about any configuration that we fail to set back to original.
```

Comment:
```suggestion
  - Before my internship, I read about how Pulumi provides a platform for building infrastructure. At first, I thought the benefit would just be to reduce the amount of work to click through different settings on the provider's website. However, during the development process of CMS, I found writing infrastructure as code can do many more things than that. For example, I have implemented Github Actions, which will deploy the infrastructure automatically when I push or merge a pull request. It saves time to upload and deploy again and prevent any manual error by automating the process. Also, if we want to completely remove the cloud resources, Pulumi enables us to do a simple `pulumi destroy` rather than manually going through and worrying about missing any resources.
```